js/src/personal/riyuan/components/about/About.tsx

```diff
@@ -37,7 +37,16 @@ function InfoSection({ title, description, listItems }: InfoSectionProps) {
         size="sm"
       >
         {listItems.map((item, index) => (
```

Comment:
nit: should avoid using loop index as key. Not an issue now because listItems does not change ever IIUC.

Reply:
fix later